dataeng/jobs/analytics/AggregateDailyTrackingLogs.groovy

```diff
@@ -12,6 +12,7 @@ class AggregateDailyTrackingLogs {
     public static def job = { dslFactory, allVars ->
         allVars.get('ENVIRONMENTS').each { environment, env_config ->
             dslFactory.job("aggregate-daily-tracking-logs-$environment") {
+                description("This job copys compressed traking logs of daily aggreagates from source bucket to destination bucket with a pattern of '*.tracking.log-date.gz'.")
```

Comment:
Change to: `This job aggregates tracking log files by grouping them by date, producing the aggregated output in the destination bucket.`